kinetic/distribution.yaml

```diff
@@ -273,6 +273,21 @@ repositories:
       url: https://github.com/ros-perception/image_common.git
       version: hydro-devel
     status: maintained
+  lyap_control:
+    doc:
+      type: git
+      url: https://bitbucket.org/AndyZe/lyap_control.git
+      version: master
+    release:
+      tags:
+        release: release/indigo/{package}/{version}
```

Comment:
You can't just use the Indigo release. You need to run bloom for the Kinetic for your repo.